### PR TITLE
Homepage Improvements + more

### PR DESCRIPTION
**Pull request by pedroCX486**
In this PR...

Some changes to the landing page:

CleanShot 2024-01-31 at 03 16 31

CleanShot 2024-01-31 at 03 16 41

Changes to the warning for Mac users:

CleanShot 2024-02-04 at 16 23 21

And some general code cleanup, removing some shoddy CSS I left behind and changing some messaging around.

### CODE REVIEW

File: src/app/page.tsx [hunk 1]
```diff
@@ -35,19 +35,37 @@ const Home = () => {
...
-              <p>
-                Looks like you are using a Mac. If you come across any issues,
-                please let us know{" "}
-                <a
-                  className="text-blue-400 underline"
-                  href="https://github.com/portapack-mayhem/MayhemHub/issues/43"
-                  target="_blank"
-                  rel="noopener noreferrer"
+              <div role="alert" className="alert alert-warning m-5 w-[70%]">
+                <svg
```
Comment (jLynx) [inline src/app/page.tsx:39]: Can we swap this for a FA icon?

File: src/app/page.tsx [hunk 2]
```diff
@@ -35,19 +35,37 @@ const Home = () => {
...
+                  <path
+                    strokeLinecap="round"
+                    strokeLinejoin="round"
+                    strokeWidth="2"
+                    d="M12 9v2m0 4h.01m-6.938 4h13.856c1.54 0 2.502-1.667 1.732-3L13.732 4c-.77-1.333-2.694-1.333-3.464 0L3.34 16c-.77 1.333.192 3 1.732 3z"
+                  />
+                </svg>
+                <span>
+                  <p>
+                    <p className="text-2xl">Whoa there, cowpoke!</p>
```
Comment (jLynx) [inline src/app/page.tsx:54]: Please remove this title as its not needed

File: src/app/page.tsx [hunk 3]
```diff
@@ -35,19 +35,37 @@ const Home = () => {
...
             {isMac() && (
-              <p>
-                Looks like you are using a Mac. If you come across any issues,
-                please let us know{" "}
-                <a
-                  className="text-blue-400 underline"
-                  href="https://github.com/portapack-mayhem/MayhemHub/issues/43"
-                  target="_blank"
-                  rel="noopener noreferrer"
+              <div role="alert" className="alert alert-warning m-5 w-[70%]">
```
Comment (jLynx) [inline src/app/page.tsx:38]: Could we please have the same background colour as the home page text items

**Review by jLynx — CHANGES_REQUESTED** [review]
Just a few small things around the mac warning. The rest looks really good

File: src/app/page.tsx (hunk 1 above)
Comment (pedroCX486) [reply src/app/page.tsx:39]: Ah, yep! I just copied the DaisyUI example 🫣

**Review by jLynx — APPROVED** [review]
Looks great!

**Pull request merged**